src/FullBuild/Program.fs

```diff
@@ -49,12 +49,12 @@ let tryMain argv =
     // view
     | Command.AddView viewInfo -> Commands.View.Add viewInfo
     | Command.DropView viewInfo -> Commands.View.Drop viewInfo.Name
-    | Command.ListViews -> Commands.View.List ()

```

Comment:
3 solution -
1. Dont do it
2. Scan it with index command and put to some central repo
3. Scan all dirs for *.fbv files on list command
I prefer the first one :) Will discuss later